extra/linux/redhat/alacritty.spec

```diff
@@ -0,0 +1,56 @@
+Name:          alacritty
+Version:       0.3.2
+Release:       1%{?dist}
+Summary:       A cross-platform, GPU enhanced terminal emulator
+License:       ASL 2.0
+URL:           https://github.com/jwilm/alacritty
+VCS:           https://github.com/jwilm/alacritty.git
+Source:        alacritty-%{version}.tar
+
+BuildRequires: rust >= 1.31.0
```

Comment:
This needs to be changed to 1.32.0, since I've just updated the minimum supported rust version.

Reply:
Done, currently build new packages for testing